webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -39,7 +39,14 @@ class BookmarkContainer extends React.Component {
     ) : (
       <Dropdown pullRight id={controller}>
         <Dropdown.Toggle />
-        <Dropdown.Menu>
+        <Dropdown.Menu className='scrollable-dropdown'>
+          {canCreate && (
+            <MenuItem key="newBookmark" id="newBookmark" onClick={this.onClick.bind(this)}>
+              {__('Bookmark this search')}
+            </MenuItem>
+          )}
+          <DocumentationUrl id="bookmarkDocumentation" href={this.props.documentationUrl} />
```

Comment:
`documentationUrl` can be extracted from props at the beginning of this function.